docs/baseband/mark5b/index.rst

```diff
@@ -22,19 +22,19 @@ Each :term:`data frame` consists of a :term:`header` consisting of four 32-bit
 words (16 bytes) followed by a :term:`payload` of 2500 32-bit words (10000
 bytes).  The header contains a sync word, frame number, and timestamp 
 (accurate to 1 ms), as well as user-specified data; see Sec. 1 of the
-`specifications <m5bspec>`_ for details.  The payload supports $2^n$
-1-bit-streams, for $0 \leq n \leq 5$, and the first sample of each stream
-corresponds precisely to the header time.  |Elementary samples| may be 1 or 2
-bits in size, with the latter being stored in two successive bit streams.  The
-number of |channels| is equal to the number of bit-streams
-divided by the number of bits per elementary sample (Baseband currently only
-supports files where all bit-streams are active).  Files begin at a header 
-(unlike for Mark 4), and an integer number of frames fit within 1 second.
+`design specifications`_ for details.  The payload supports $2^n$ 1-bit-streams,
+for $0 \leq n \leq 5$, and the first sample of each stream corresponds
+precisely to the header time.  |Elementary samples| may be 1 or 2 bits in size,
+with the latter being stored in two successive bit streams.  The number of
+|channels| is equal to the number of bit-streams divided by the number of bits
+per elementary sample (Baseband currently only supports files where all
+bit-streams are active).  Files begin at a header (unlike for Mark 4), and an
+integer number of frames fit within 1 second.
 
 The Mark 5B system also outputs files with the active bit-stream mask, number
-of frames per second, and observational metadata (Sec. 1.3 of the
-`specifications <m5bspec>`_).  Baseband does not use these files (instead
-requiring the user specify, for example, the :term:`sample rate`).
+of frames per second, and observational metadata (Sec. 1.3 of the `design
+specifications`_).  Baseband does not use these files (instead requiring the
```

Comment:
Maybe "does not yet use these files (as they seem rarely included in data sets), instead ..."

Reply:
Not sure we need to explain why this isn't supported, so left out the parenthesis.